posts/2020-11-26-beta-release.adoc

```diff
@@ -0,0 +1,215 @@
+
+The above definition will result in the files /bindings/account_db/username and /bindings/account_db/password being created when the pod is created. Liberty will create variables from both files, and users can access them using normal Liberty variable syntax, `${account_db/username}` and `${account_db/password}`.
+
+For more Information:
+* link:https://github.com/k8s-service-bindings/spec[Service Binding specification]
+* link:https://kubernetes.io/docs/concepts/configuration/secret/[Kubernetes Secrets]
+
+=== Try it now 
+
+To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 15, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+  <groupId>io.openliberty.beta</groupId>
+  <artifactId>openliberty-runtime</artifactId>
+  <version>21.0.0.1-beta</version>
+  <type>pom</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-runtime', version: '[21.0.0.1-beta,)'
+}
+----
+
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+The main change visible to developers in Jakarta EE is the names of packages changing to accomodate the new `jakarta.*` namespace. In this Open Liberty beta, we have more ... API Release Candidates.
+// // // // // // // //
+// ABOVE:
+// Change the "..." to something describing what's new at a high level. 
+// // // // // // // //
+
+This Open Liberty beta introduces the following Jakarta EE 9 features which now possess their all-new Jakarta EE 9 package names:
+
+* <<face, Jakarta Faces Container 3.0 (`facesContainer-3.0`)>>
+* <<timer, Jakarta Enterprise Beans Persistent Timer 4.0 (`ejbPersistentTimer-4.0`)>>
+
+These join the Jakarta EE 9 features in link:https://openliberty.io/blog/?search=beta&key=tag[previous Open Liberty betas]:
+
+* Jakarta WebSocket 2.0 (`websocket-2.0`; now with full CDI integration)
+* Enterprise Beans Remote Client 2.0 (`ejbRemoteClient-2.0`)
+* RESTful Web Services 3.0 (`restfulWS-3.0` and `restfulWSClient-3.0`)
+* Jakarta Server Faces 3.0 (`faces-3.0`)
+* Jakarta Connectors 2.0 (`connectors-2.0`)
+* Jakarta Enterprise Beans Remote 4.0 (`ejbRemote-4.0`)
+* Jakarta EE Application Client 9.0 (`javaeeClient-9.0`)
+* Jakarta Authentication 2.0 (`jaspic-2.0`)
+* Jakarta Authorization 2.0 (`jacc-2.0`)
+* Jakarta Persistence 3.0 (includes Eclipselink 3.0-RC1.) (`jpa-3.0`)
+* Jakarta XML Binding 3.0 (`jaxb-3.0`)
+* Jakarta Managed Beans 2.0 (`managedBeans-2.0`)
+* Jakarta Concurrency 2.0 (`concurrent-2.0`)
+* Jakarta Enterprise Beans Home 4.0 (`ejbHome-4.0`)
+* Jakarta Enterprise Beans Lite 4.0 (`ejbLite-4.0`)
```

Comment:
Change "ejbHome-4.0" to "enterpriseBeansHome-4.0"
Change "ejbLite-4.0" to "enterpriseBeansLite-4.0"
the "ejb" versions do not exist.

Reply:
Also, need to add:
"Jakarta Enterprise Beans 4.0 (`enterpriseBeans-4.0`)
Not sure why this feature isn't already listed; it has been in the betas for awhile.

Perhaps move all the "enterpriseBeansxxx" feature together in this list.

Reply:
Done